style.css

```diff
@@ -540,3 +544,221 @@ span {
+  }
+
+  #navigator_bar .internal-link li a {
+    text-decoration: none;
+    font-family: poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .starting-page {
+    height: 100vh;
+    background-image: url("assets/Group\ 58.png");
+    background-repeat: no-repeat;
+    background-size: 90%;
+  }
+
+  #section-presentation {
+    height: 916px;
+    margin-left: 18%;
+    margin-right: 18%;
+    margin-top: 200px;
+    justify-content: stretch;
+    background-image: none;
+  }
+
+  #my_work {
+    margin-top: 100px;
+    display: grid;
+    grid-template-rows: repeat(18, 100vh);
+  }
+
+  .general {
+    grid-column: 2 / 1 span;
+    align-self: center;
+  }
+
+  .project {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    width: 80%;
+    height: 90%;
+    margin-left: 10%;
+    grid-column-gap: 20px;
+    margin-bottom: 100px;
+  }
+
+  #general_inf {
+    grid-column: 2 / 1 span;
+  }
+
+  .project footer .external_link {
+    margin-left: 0;
+    width: 18%;
+    padding: 2% 0;
+  }
+
+  .project_image {
+    grid-column: 1 / 1 span;
+
+    /* max-width: 100%;
+    min-height: 70%;
+    max-height: 411px; */
+    width: 95%;
+    justify-self: center;
+    margin-left: 16px;
+    align-self: center;
+  }
+
+  #about-myself {
+    display: grid;
+    grid-template-columns: 40% 60%;
+    margin: 0 8% 10%;
+    grid-column-gap: 5%;
+  }
+
+  .myself-abt {
+    grid-column: 1 / span 1;
+    margin-right: 0;
+  }
+
+  .myself-abt h1 { margin-bottom: 0; }
+
+  #resume-link h2 { margin-bottom: 0; }
+
+  #resume-link ul { margin-top: 8px; }
+
+  #presentation { margin-bottom: 0; }
+
+  #my_skills {
+    grid-column: 2 / span 1;
+    margin-top: 156px;
+    margin-left: 0;
+    width: 100%;
+    height: 300px;
+  }
+
+  .spec_name {
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  #programmes {
+    flex-direction: row;
+    justify-content: space-between;
+    margin-right: 0;
+  }
+
+  .tech-o {
+    flex-direction: column;
+    justify-content: flex-start;
+    height: 100px;
+    width: 102px;
+    align-items: center;
+  }
+
+  .tech-o p {
+    font-family: Poppins, sans-serif !important;
+    font-style: normal !important;
+    font-weight: 500 !important;
+    font-size: 15px !important;
+    line-height: 20px !important;
+    color: #253858 !important;
+    margin-left: 15px !important;
+    margin-top: 15px !important;
+  }
+
+  .tech-o img { align-self: flex-start; }
+
+  #l1 hr {
+    background: #dfe1e6;
+    height: 1px;
+  }
+
+  #sivak { justify-content: space-between; }
+
+  .wrap {
+    justify-content: center;
+    width: 50%;
+    height: 25%;
+    margin-bottom: 200px;
+  }
+
+  .form-section {
+    background-image: url("assets/contact\ form\ background\ shapes.png");
+    background-repeat: no-repeat;
+    background-color: #616efa;
+    background-size: 85%;
+    background-position: center;
+    border-radius: 7% 0 0 0;
+  }
```

Comment:
- [ ] As I mentioned above :point_up: you must follow the Figma Design but your form is not the same as Figma Design kindly make your design the same as much as possible in the  Figma design check below :point_down:  comparison screenshots :+1: 

**Your Design**
![Screenshot from 2023-02-23 22-13-56](https://user-images.githubusercontent.com/45226848/220989610-15279776-6d48-4865-b51e-f2f9375582c7.png)

**Figma Design**
![Screenshot from 2023-02-23 22-13-39](https://user-images.githubusercontent.com/45226848/220989655-06651985-e86c-4223-b67c-8644edf1101f.png)

Reply:
Thank you for notifying and helping me with the guidance. I have resolved the changes in the `contact me` box as mentioned.